README.md

```diff
@@ -106,3 +106,18 @@ Stop the local postgres:
 ```
 local-dev/run_postgres.sh stop
 ```
+## Authorization
+Janitor uses a list of user email address as administrator user to access its admin endpoint.
+The value can be set in `iam.adminUserList` configuration.
+### For Broad Engineers:
+The file is stored in Vault. 
+To read user list, run:
+```
+docker run -e VAULT_TOKEN=$(cat ~/.vault-token) -it broadinstitute/dsde-toolbox:dev vault read config/terra/crl-janitor/common/iam 
+```
+To request admin access, please contact [mc-terra-janitor-admins](https://github.com/orgs/broadinstitute/teams/mc-terra-janitor-admins) members.
```

Comment:
The admin list has permission to modify the Vault value. So this section's header is "for broad engineer", because this is where Broad Janitor stores the config value. Other Janitor can have there own way to store the config value, no functional impact